test/unit/testDeceasedName.js

```diff
@@ -14,25 +14,41 @@ describe('DeceasedName', () => {
         });
     });
 
-    describe('handlePost()', () => {
-        let ctx;
-        let errors;
-        let formdata;
-        let session;
-        let hostname;
-        let featureToggles;
-
-        it('should return the ctx with the deceased name and the screening_question feature toggle', (done) => {
-            ctx = {
+    describe('getContextData()', () => {
+        it('should return the ctx with the deceased address and the screening_question feature toggle on', (done) => {
```

Comment:
Please update the description for this test as it is for deceased name but it is referring to deceased address.